### ISSUE TITLE
Some internal links are broken.

### ISSUE THREAD

**Issue by orzih** [squidfunk/mkdocs-material#1901]
I am creating a PDF generation plugin.
I checked with your "docs", and found some broken links.
I think below:

```diff
diff -r ORIG/docs/reference/lists.md docs/reference/lists.md
174c174
<   [6]: #definition-lists
---
>   [6]: #definition-list
diff -r ORIG/docs/setup/setting-up-site-analytics.md docs/setup/setting-up-site-analytics.md
13c13
<   [2]: #using-other-analytics-services
---
>   [2]: #other-analytics-providers
```

Are there right?

**Comment by squidfunk** [comment]
Feel free to create a PR! Those should link to the respective sections on the same site.

**Comment by orzih** [comment]
Thanks, I will be PR later.

**Comment by squidfunk** [comment]
Fixed in 38d054b7.
